docs/src/complexity_measures.md

```diff
@@ -11,3 +11,17 @@ complexity_normalized
 ReverseDispersion
 distance_to_whitenoise
 ```
+
+## Approximate entropy
+
+```@docs
+ApproxEntropy
```

Comment:
Can we name this `ApproximateEntropy` directly? I'ts already long eniough where saving the extra 5 characters doesn't matter.

Reply:
Yes, why not.